rv32zild_proposal.adoc

```diff
@@ -47,4 +47,4 @@ If the compressed extension link:++https://github.com/riscv/riscv-code-size-redu
 
 This is of course incompatible with the (RV32) F extension, as those opcodes are then occupied by the single-precision load/store ops (C.FLW, C.FSW, C.FLWSP and C.FSWSP respectively).
 Therefore those instructions should be added as a separate *Zcld* or similar sub-extension (or naturally follow if *Zild* and *Zca* [but _not Zcf_] are enabled). +
-Because of the <<restrictions>> imposed on register selection, 1 bit of all the compressed encodings (the ones for _rs2[0]_ and _rd[0]_) can be reserved and thus we free up half the code points for future use.
+Because of the <<restrictions>> imposed on register selection, 2 bits of all the compressed encodings (the ones for _rs2[0]_ and _rd[0]_) can be reserved and thus we free up 3/4 of the code points for future use. 
```

Comment:
I initially thought it could save 2 bits as well, but it only saves 1 bit for each encoding (rs1 still uses all the selection bits, so there's only one register specifier in each instruction that's affected by the addressing limitation).

Otherwise the suggested changes look good!